src/api/api_models.rs

```diff
@@ -148,6 +148,23 @@ impl WithImages for Artist {
     }
 }
 
+#[derive(Deserialize, Debug, Clone)]
+pub struct User {
+    pub id: String,
+    pub display_name: String,
+    pub images: Option<Vec<Image>>,
```

Comment:
(not being used for now unless I'm missing something?)

Reply:
It's used as the return type for get user `SpotifyClient::get_client` https://github.com/Douile/spot/blob/098b62962a60d8491212d02ecc190e67a4d34af3/src/api/client.rs#L369-L373

Reply:
Oh I was thinking specifically about the images!

Reply:
Ah, okay mis-understood you

Reply:
Had it cos thinking of having user image, but left it out to keep consistent with the artist view. What do you think of adding profile image @xou816 ?